lte/cloud/go/services/lte/lte/main.go

```diff
@@ -47,8 +48,13 @@ func main() {
 		glog.Infof("Failed unmarshalling service config %v", err)
 		return
 	}
+
+	// initialize analytics
+	// userStateExpr is a metric which enables us to compute the number of active users using the network
+	promQLClient := analytics.GetPrometheusClient()
 	protos.RegisterAnalyticsCollectorServer(srv.GrpcServer,
```

Comment:
Use the golang style for multiline functions etc

```
protos.RegisterAnalyticsCollectorServer(
    srv.GrpcServer,
    analytics.NewCollectorService(
        &serviceConfig.Analytics,
        promQLClient,
        lte_analytics.GetAnalyticsCalculations(&serviceConfig.Analytics),
        calculations.NewUserStateMgr(promQLClient, "ue_connected")
    )
)
```

Though here it would read better if the NewCollectorService was created on a preceding line, as its own variable